src/form-elements/FormElementCamera.tsx

```diff
@@ -88,43 +88,44 @@ function FormElementCamera({
-                const context = canvas.getContext('2d')
-                if (context) {
-                  const now = localisationService.formatDatetime(
-                    new Date(file.lastModified),
-                  )
-                  const textHeight = 20
-                  context.font = `${textHeight}px Arial`
-                  const { width: textWidth } = context.measureText(now)
-                  const backgroundMargin = 10
-                  const backgroundPadding = backgroundMargin
-                  const backgroundWidth = backgroundPadding * 2 + textWidth
-                  const backgroundHeight = backgroundPadding * 2 + textHeight
-                  context.fillStyle = 'rgba(20, 20, 20, 0.6)'
-                  context.fillRect(
-                    canvas.width - backgroundWidth - backgroundMargin,
-                    canvas.height - backgroundHeight - backgroundMargin,
-                    backgroundWidth,
-                    backgroundHeight,
-                  )
-
-                  context.fillStyle = '#FFF'
-                  context.fillText(
-                    now,
-                    canvas.width -
-                      textWidth -
-                      backgroundPadding -
-                      backgroundMargin,
-                    canvas.height - 22,
-                  )
-                }
-              }
               let base64data
               try {
+                let context
+                if (element.includeTimestampWatermark) {
+                  context = canvas.getContext('2d')
+                  if (context) {
+                    const now = localisationService.formatDatetime(
+                      new Date(file.lastModified),
+                    )
+                    const textHeight = 20
+                    context.font = `${textHeight}px Arial`
+                    const { width: textWidth } = context.measureText(now)
+                    const backgroundMargin = 10
+                    const backgroundPadding = backgroundMargin
+                    const backgroundWidth = backgroundPadding * 2 + textWidth
+                    const backgroundHeight = backgroundPadding * 2 + textHeight
+                    context.fillStyle = 'rgba(20, 20, 20, 0.6)'
+                    context.fillRect(
+                      canvas.width - backgroundWidth - backgroundMargin,
+                      canvas.height - backgroundHeight - backgroundMargin,
+                      backgroundWidth,
+                      backgroundHeight,
+                    )
+
+                    context.fillStyle = '#FFF'
+                    context.fillText(
+                      now,
+                      canvas.width -
+                        textWidth -
+                        backgroundPadding -
+                        backgroundMargin,
+                      canvas.height - 22,
+                    )
+                  }
+                }
                 base64data = canvas.toDataURL(file.type)
```

Comment:
wont this error because context will be null if `context = canvas.getContext('2d')` doesnt find one?